services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -218,6 +219,63 @@
     db_engine: Engine, project_id: str, tasks: Dict[str, Dict], replace_pipeline=True
 ):
     # pylint: disable=no-value-for-parameter
+
+    async def _task_already_exists(
+        conn: SAConnection, project_id: str, node_id: str
+    ) -> bool:
+        task_count: int = await conn.scalar(
+            sa.select([sa.func.count()]).where(
+                and_(
+                    comp_tasks.c.project_id == project_id,
+                    comp_tasks.c.node_id == node_id,
+                )
+            )
+        )
+        assert task_count in (
+            0,
+            1,
+        ), f"Uniqueness violated: task_count={task_count}"  # nosec
```

Comment:
sure about that one? it will be removed -> see codeclimate issue

Reply:
Note the ``#nosec``. This is just as doc of what I would expect and only used during development. Removed in production. See https://wiki.c2.com/?AssertionsAsDefensiveProgramming